### DISCUSSION TITLE
Stream and ESP-Now

### DISCUSSION THREAD

**Discussion by grender007** [pschatzmann/arduino-audio-tools#1463] [Q&A] [answered]
Hi!
I'm trying to communicate two ESP32-S3 using ESP-Now. One of the microcontrollers (the "sender") gets the ouput of an audio codec as a Stream variable, but I would like to know if this library is compatible with ESP-NOW or if I need to make some modifications.

**Comment by pschatzmann** [comment, marked as answer]
Just look at the [documentation](https://github.com/pschatzmann/arduino-audio-tools/wiki/Communication) and the examples
